tests/conftest.py

```diff
@@ -57,3 +58,18 @@ def fake_fs():
         patcher.fs.add_real_directory(test_data_path)
 
         yield patcher.fs # This will return a fake file system
+
+
+def set_home(monkeypatch, path):
```

Comment:
Interesting that you don't need to change the direction of the slashes here?

Reply:
I didn't find anything on internet regarding this. But maybe it fixes the path before setting the variable.